cpp/src/endpoint.cpp

```diff
@@ -543,14 +553,15 @@ std::shared_ptr<Worker> Endpoint::getWorker() { return ::ucxx::getWorker(_parent
 void Endpoint::errorCallback(void* arg, ucp_ep_h ep, ucs_status_t status)
 {
   ErrorCallbackData* data = reinterpret_cast<ErrorCallbackData*>(arg);
-  data->status            = status;
+  if (data->closing->exchange(true)) return;
```

Comment:
OK, so this is the third place that a close request can be emitted?

Reply:
Yes, but this cannot be controlled by the user.